int64 left_damping
int64 right_damping
---
int8 result
